src/sdk/namespace_fs.js

```diff
@@ -2386,7 +2388,11 @@ class NamespaceFS {
         try {
             await nb_native().fs.stat(fs_context, this.bucket_path);
         } catch (err) {
-            throw this._translate_object_error_codes(err);
+            dbg.warn('_load_bucket failed, on bucket_path', this.bucket_path, 'got error', err);
+            const err_to_throw = this._translate_object_error_codes(err);
```

Comment:
With the change of the mutual function, you will just need to call it with 'BUCKET' entity instead of doing this manual change.

Reply:
I changed it, thanks.